tests/src/Kernel/Entity/EntityGroupsTest.php

```diff
@@ -0,0 +1,216 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\Tests\og\Kernel\Entity\EntityGroupsTest.
+ */
+
+namespace Drupal\Tests\og\Kernel\Entity;
+
+use Drupal\entity_test\Entity\EntityTest;
+use Drupal\KernelTests\KernelTestBase;
+use Drupal\Component\Utility\Unicode;
+use Drupal\og\Entity\OgMembership;
+use Drupal\og\Og;
+use Drupal\user\Entity\User;
+
+/**
+ * Tests entity group methods
+ *
+ * @group og
+ */
+class EntityGroupsTest extends KernelTestBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public static $modules = ['system', 'user', 'field', 'og', 'entity_test'];
```

Comment:
I don't think that will happen in kernel tests, I think you need to be explicit about the modules and schemas you want available.